src/test/scala/akka/contrib/process/BlockingProcessSpec.scala

```diff
@@ -98,12 +96,9 @@ object Receiver {
 }
 
 class Receiver(probe: ActorRef, command: String, stdinInput: immutable.Seq[String], nameSeed: Long) extends Actor
-    with Stash
-    with ImplicitMaterializer {
+    with Stash with ImplicitMaterializer {
```

Comment:
It's for storing all the messages when the merged `stdout` and `stderr` stream are not completed:

https://github.com/typesafehub/akka-contrib-extra/pull/46/files#diff-ac1a75a9b3c83841178d1c3d7d22c210L128

Once completed, all the messages will be replayed using `unstashAll()`

https://github.com/typesafehub/akka-contrib-extra/pull/46/files#diff-ac1a75a9b3c83841178d1c3d7d22c210L123